kroxylicious-kms-provider-hashicorp-vault/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/config/Config.java

```diff
@@ -14,18 +14,19 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import io.kroxylicious.proxy.config.tls.PasswordProvider;
```

Comment:
would be good to move this to a different package, doesn't need to be part of this PR

Reply:
Done, I moved them to '.secret'.